src/chart.js

```diff
@@ -12,14 +12,14 @@ Chart.defaults = require('./core/core.defaults');
 Chart.Element = require('./core/core.element');
 Chart.elements = require('./elements/index');
 Chart.Interaction = require('./core/core.interaction');
+Chart.layout = require('./core/core.layout');
```

Comment:
actually, I'm just wondering now, do we actually need a `Chart.layout` variable? shouldn't folks just call `require('./core/core.layout')` when they need the layout?

Reply:
The layout (singleton) can be accessed from external plugins/chart, in which case there is no possibility to require `'./core/core.layout'`.

Reply:
@etimberg @benmccann Actually, I don't know why the layout is a singleton instead of an object attached to each chart and accessible via `chartInstance.layout`. Maybe that something we will like to change in v3.

For the alias, I would suggest another name to be consistent with other "Service" singletons: `Chart.layouts` (plural since it manages many layouts), as we have `Chart.plugins` and soon `Chart.animations` and `Chart.scales`.

What do you think?